### PR TITLE
332 fix welcome-screen login button

### LINKED ISSUE #332: fix welcome-screen login button
it''s jsut to small and does not look right

### PR DESCRIPTION
**Pull request by JanEberlein**
closes #332 

now looks like this:
image
image

### CODE REVIEW

**Review by tcoro — APPROVED** [review]
luvit

File: packages/app/src/components/uiBlocks/Button.vue [hunk 1]
```diff
@@ -10,7 +10,8 @@
         class="my-auto font-heading font-extralight"
         :class="{
           'ml-4': icon,
-          'mx-auto': (!rounded && !icon) || textCenter,
+          'mx-auto': !rounded && !icon,
+          'w-full text-center': textCenter,
```
Comment (schnickalodeon) [inline packages/app/src/components/uiBlocks/Button.vue:14]: Did you checked if that changes the style of all the other buttons?

**Review by schnickalodeon — APPROVED** [review]
Please check if that changes the style of all the other buttons?

File: packages/app/src/components/uiBlocks/Button.vue (hunk 1 above)
Comment (anbraten) [reply packages/app/src/components/uiBlocks/Button.vue:14]: seems to be fine

**Pull request merged**